.rubocop.yml

```diff
@@ -50,3 +50,6 @@ Style/CaseEquality:
 Style/ClassAndModuleChildren:
   Exclude:
   - "spec/tapioca/**/*"
+
+Sorbet/BindingConstantWithoutTypeAlias:
+  Enabled: false
```

Comment:
This is meant to be temporary and is related to: https://github.com/Shopify/rubocop-sorbet/issues/104